modulus/experimental/resdiff/training/loss.py

```diff
@@ -204,14 +203,14 @@ def __init__(self, P_mean=0.0, P_std=1.2, sigma_data=0.5):
         self.P_std = P_std
         self.sigma_data = sigma_data
         
-        print('dist.get_rank', dist.get_rank)
-        if dist.get_rank == 0:
-            breakpoint()
+        # print('dist.get_rank', dist.get_rank)
```

Comment:
Is this suppose to be still commented out? Just checking.

Reply:
This is debugging code so it's safe t comment it out